apptest_test.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 func TestTestStringReplacements(t *testing.T) {
-	d, _, h := prepareTestChain("test")
-	cleanupTestDir(d)
+	cleanup, _, h := prepareTestChain("test")
+	cleanup()
```

Comment:
This is how the tests clean up after themselves. They return a func() the you call to cleanup, usually with defer, but this case was using it to create a dir that is empty for the test case.